public/sitemap-0.xml

```diff
@@ -0,0 +1,47 @@
+<url><loc>https://cow.fi/tokens/flux-frax</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/flux-usdc</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/flux-usdt</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/frax</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/frax-share</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/gitcoin</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/giveth</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/hex</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/honey</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/liquity</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/liquity-usd</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/magic-internet-money</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/manifold-finance</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/matic-network</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/metaverse-index</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/origin-ether</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/reflexer-ungovernance-token</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/ribbon-finance</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+<url><loc>https://cow.fi/tokens/rocket-pool</loc><lastmod>2023-07-06T17:57:36.255Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
+</urlset>
```

Comment:
do we need to commmit this file in the project? if not we should delete and add to .gitignore

Reply:
Thought about it too. In principle  it is dynamic of nature so therefore I would indeed to gitignore